syntax = "proto2";
package Qot_GetBroker;

import "Common.proto";
import "Qot_Common.proto";

message C2S
{
	required Qot_Common.Security security = 1; //股票
}

message S2C
{
	required Qot_Common.Security security = 1; //股票
	repeated Qot_Common.Broker brokerAskList = 2; //经纪Ask(卖)盘
	repeated Qot_Common.Broker brokerBidList = 3; //经纪Bid(买)盘
}

message Request
{
	required C2S c2s = 1;
}

message Response
{
	required int32 retType = 1 [default = -400]; //RetType,返回结果
	optional string retMsg = 2;
	optional int32 errCode = 3;
	optional S2C s2c = 4;
}
